src/main/scala/com/gu/fulfilmentLookup/Lambda.scala

```diff
@@ -66,16 +66,16 @@ trait FulfilmentLookupLambda extends Logging {
         val subNames = rows.map(row => row.subscriptionName)
         val subInFile = subNames.contains(lookupRequest.subscriptionName)
         val subIndex = subNames.indexOf(lookupRequest.subscriptionName)
-        val responseBody = LookupResponseBody(
+        val result = LookupResult(
           fileName,
           subInFile,
           populateAddressRecord(rows, subIndex)
         )
         logger.info(s"Performed successful lookup for ${lookupRequest.subscriptionName} in $fileName. subInFile: $subInFile | subIndex: $subIndex")
-        raiseCase.open(config, lookupRequest, responseBody) match {
+        raiseCase.open(config, lookupRequest, result) match {
           case \/-(_) =>
             logger.info("Successfully raised Salesforce case")
-            LookupResponse(200, responseBodyAsString(responseBody))
+            LookupResponse(200, responseBodyAsString(LookupResponseBody(result.fileChecked, result.subscriptionInFile)))
```

Comment:
we don't return the address any more, but the other fields are also ignored on the client side?

Reply:
I left these for now because I thought we might want to display slightly different messaging based on whether or not the sub was in the file.